google-cloud-error_reporting/docs/authentication.md

```diff
@@ -0,0 +1,43 @@
+## With `google-cloud-ruby`
+
+With `google-cloud-ruby` it's incredibly easy to get authenticated and start using Google's APIs. You can set your credentials on a global basis as well as on a per-API basis.
+
+### Project and Credential Lookup
+
+Gcloud aims to make authentication as simple as possible, and provides several mechanisms to configure your system without providing **Project ID** and **Service Account Credentials** directly in code.
```

Comment:
@hxiong388 Makes sense. This needs to be fixed everywhere, so I will do it in a new PR once this one is merged. Thanks @dazuma for commenting on this.